apps/libs/rootserver/testing/ApiTesting.ts

```diff
@@ -0,0 +1,46 @@
+import { NestFastifyApplication } from '@nestjs/platform-fastify'
+import { InjectOptions, Response as LightMyRequestResponse } from 'light-my-request'
+import { RootServer } from '@defichain-apps/libs/rootserver'
+
+/**
+ * Testing framework.
+ */
+export abstract class ApiTesting {
+  protected abstract stubServer: StubServer
```

Comment:
You may try something like this.

```suggestion
  protected abstract stubServer: RootServer
```